pkgs/development/libraries/proj/default.nix

```diff
@@ -34,6 +34,10 @@ stdenv.mkDerivation rec {
   ];
 
   doCheck = true;
+  # these tests fail on darwin hydra but nowhere else
```

Comment:
That's apparently not true: https://github.com/NixOS/nixpkgs/issues/142875#issuecomment-963735898